### PR TITLE
Use api instead of dynamic html

### PR DESCRIPTION
**Pull request by henneboy**
- Created active products api
- Remove dynamic html generation

### CODE REVIEW

File: src/protocol/products/active_products_response.rs [hunk 1]
```diff
@@ -0,0 +1,15 @@
+use serde::{Deserialize, Serialize};
+
+use crate::dso::product::ProductId;
+
+#[derive(Deserialize, Serialize)]
+pub struct ActiveProductsReponse {
```
Comment (mads256h) [inline src/protocol/products/active_products_response.rs:6]:
```suggestion
pub struct ActiveProductsResponse {
```

**Review by mads256h — CHANGES_REQUESTED** [review]
Add quick buy call

**Comment by henneboy** [comment]
@mads256h plz help, github desktop can't reset a branch and I'm not a git user. I cannot fix this without fork...

**Comment by henneboy** [comment]
@Casper-NS || @miniJmo plz review, thx :)

**Review by mads256h — APPROVED** [review]

**Pull request merged**